packages/convertim/src/Model/Order/ConvertimOrderDataToOrderDataMapper.php

```diff
@@ -0,0 +1,192 @@
+            $orderData->postcode = $convertimBillingAddressData->getPostcode();
+            $orderData->country = $this->getCountryByName($convertimBillingAddressData->getCountry());
+        } else {
+            $orderData->companyName = $convertimDeliveryAddressData->getCompanyName();
+            $orderData->street = $convertimDeliveryAddressData->getStreet();
+            $orderData->city = $convertimDeliveryAddressData->getCity();
+            $orderData->postcode = $convertimDeliveryAddressData->getPostcode();
+            $orderData->country = $this->getCountryByName($convertimDeliveryAddressData->getCountry());
+        }
+
+        $orderData->deliveryAddressSameAsBillingAddress = $convertimOrderData->isBillingAddressSameAsDeliveryAddress();
+
+        if (!$convertimOrderData->isBillingAddressSameAsDeliveryAddress()) {
+            $orderData->deliveryFirstName = $convertimDeliveryAddressData->getName();
+            $orderData->deliveryLastName = $convertimDeliveryAddressData->getLastName();
+            $orderData->deliveryCompanyName = $convertimDeliveryAddressData->getCompanyName();
+            $orderData->deliveryTelephone = $convertimDeliveryAddressData->getCurrierTelephoneNumberWithPrefix();
+            $orderData->deliveryStreet = $convertimDeliveryAddressData->getStreet();
+            $orderData->deliveryCity = $convertimDeliveryAddressData->getCity();
+            $orderData->deliveryPostcode = $convertimDeliveryAddressData->getPostCode();
+            $orderData->deliveryCountry = $this->getCountryByName($convertimDeliveryAddressData->getCountry());
+        }
+        $orderData->note = $convertimOrderData->getNote();
+        $orderData->heurekaAgreement = !$convertimOrderData->isDisallowHeurekaVerifiedByCustomers();
+        $orderData->origin = 'Convertim';
+
+        $this->mapProducts($convertimOrderData, $orderData);
+        $this->mapConvertimTransportOrPaymentDataToOrderItem($convertimOrderData->getPaymentData(), $orderData);
+        $this->mapConvertimTransportOrPaymentDataToOrderItem($convertimOrderData->getTransportData(), $orderData);
+
+        $this->cartFacade->deleteCart($cart);
+
+        return $orderData;
+    }
+
+    /**
+     * @param string $uuid
+     * @param \Shopsys\FrameworkBundle\Model\Order\Item\OrderItemData[] $orderItemDatas
+     * @return \Shopsys\FrameworkBundle\Model\Order\Item\OrderItemData
+     */
+    protected function getOrderItemByUuid(string $uuid, array $orderItemDatas): OrderItemData
+    {
+        foreach ($orderItemDatas as $orderItemData) {
+            if ($orderItemData->getItemUuid() === $uuid) {
+                return $orderItemData;
+            }
+        }
+
+        throw new OrderItemNotFoundException(sprintf('Order item with uuid "%s" not found.', $uuid));
+    }
+
+    /**
+     * @param string|null $countryName
+     * @return \Shopsys\FrameworkBundle\Model\Country\Country
+     */
+    protected function getCountryByName(?string $countryName): Country
```

Comment:
If there is no country, it will end with null. 

I have changed it to code. 

I do not want to throw exception as it would fail to save order, that is not the preferred way here.  So I would like to keep it as it is